components/Panel/VotePanel/Details.tsx

```diff
@@ -39,18 +41,25 @@ export function Details({
   decodedAdminTransactions,
   umipOrUppLink,
   augmentedData,
+  assertionChildChainId,
+  assertionId,
 }: VoteT) {
   const [showDecodedAdminTransactions, setShowDecodedAdminTransactions] =
     useState(false);
   const [showRawAncillaryData, setShowRawAncillaryData] = useState(false);
+  const [showRawClaimData, setShowRawClaimData] = useState(false);
 
   function toggleShowDecodedAdminTransactions() {
     setShowDecodedAdminTransactions(!showDecodedAdminTransactions);
   }
 
+  const { data: claim } = useAssertionClaim(assertionChildChainId, assertionId);
```

Comment:
just fetch individual claim data, this is indexed by assertion id, so its pretty fast, and also it will now cache by assertion and chain